packages/frontend/src/components/Dialog/styles.module.scss

```diff
@@ -154,6 +154,7 @@ $paddingVertical: 20px;
 
 .footerActionButton {
   min-width: 120px;
+  white-space: nowrap;
```

Comment:
I know we don't support narrow screens really well, and that it's the case for most other dialogs, but it's an important feature, so maybe we should make the dialog work on narrow screens?

![image](https://github.com/user-attachments/assets/8b7180c4-ddc1-4ec0-aa4d-14afea1a1288)

Reply:
That's a different issue imo, since to achieve this we have to define a different dialog width for smaller screens. Currently it's 500px if you don't define another width.

Reply:
Can be solved with media queries

Reply:
another option would be to have a sub context menu on delete, like signal has on mobile or better visible on telegram. that way the message in the confirmation can also be less confusing (currently delete for me could also be understood as only delete on one device maybe?)

Reply:
This should be resolved by https://github.com/deltachat/deltachat-desktop/pull/4806.